airflow/utils/dates.py

```diff
@@ -24,6 +24,7 @@
 from airflow.utils import timezone
 
 cron_presets = {
+    '@minutely': '* * * * *',
```

Comment:
The word "minutely" does exist, 

https://english.stackexchange.com/questions/3091/weekly-daily-hourly-minutely

Reply:
If I understood correctly the word exists, but is archaic/has double meaning [[1]](https://www.thefreedictionary.com/minutely) [[2]](https://www.merriam-webster.com/dictionary/minutely) [[3]](https://www.dictionary.com/browse/minutely?s=t)
If this is the case, I don't think double meaning is a problem here and the word just fits nice with the rest (but English is not my first language, so I'll leave the debate to natives)

I can just close the PR, but personally I think some preset for minutes is missing. It's not about the size either, It is easier to visually find on the UI, so I would not mind having a bigger string that describes It.

Would other strings be considered ?
`@every_minute`, `@all_minutes`, `@minutes`

Reply:
May I ask what kind of dag you are scheduling every minute? It sounds very overwhelming. airflow sometimes take more than a minute just to take the task from None to queue state.
Do you have something like this running in your production?

Reply:
In my particular case, some REST requests doing routine checks. Not part of any heavy process. We haven't reach the point of taking minutes to schedule/run tasks yet, only about 50 DAGs and maximum around 20 tasks, most of It running daily.

Reply:
I'm not a native speaker either, so it's difficult for me to review this change. I just noticed that this word is not common. I will wait for the opinions of other people.

Reply:
`@everyminute` if we do decide to keep it.